docs/button.md

```diff
@@ -1,5 +1,136 @@
 ---
 id: button
 title: Button
-wip: true
 ---
+
+A basic button component that should render nicely on any platform. Supports a minimal level of customization.
+
+If this button doesn't look right for your app, you can build your own button using [TouchableOpacity](touchableopacity) or [TouchableWithoutFeedback](touchablewithoutfeedback).
+
+```res
+<Button
+  onPress={onPressLearnMore}
+  title="Learn More"
+  color="#841584"
+  accessibilityLabel="Learn more about this purple button"
+/>
+```
+
+## Example
+
+```SnackPlayer name=Button%20Example
```

Comment:
Snack don't works on our website and can't.
Maybe a simple example without other component could be nice for a start, like for Text.